Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -830,11 +867,15 @@ def fetch_incidents(client, first_fetch_time, integration_instance, last_run: di
         # The count of incidents, so as not to pass the limit
         count_incidents = 0
 
+        use_get_incident_extra_data = get_multiple_incidents_extra_data_response(client, raw_incidents[0].get('incident_id')) if \
+            raw_incidents else False
+        
         for raw_incident in raw_incidents:
             incident_id = raw_incident.get('incident_id')
 
-            incident_data = get_incident_extra_data_command(client, {"incident_id": incident_id,
-                                                                     "alerts_limit": 1000})[2].get('incident')
+            incident_data = client.get_multiple_incidents_extra_data(incident_id) if use_get_incident_extra_data else \
```

Comment:
Where do we support the fact that it is possible to use the new command with multiple incidents?

Reply:
Also, this does not seem right, why do we remove the get_incident_extra_data_command that is using the client methods instead of it using the client directly?